src/protocols/kubernetes/client.c

```diff
@@ -96,7 +96,7 @@ int guac_client_init(guac_client* client) {
     client->data = kubernetes_client;
 
     /* Init clipboard */
-    kubernetes_client->clipboard = guac_common_clipboard_alloc(GUAC_KUBERNETES_CLIPBOARD_MAX_LENGTH);
+    kubernetes_client->clipboard = guac_common_clipboard_alloc();
```

Comment:
I think allocation of this should be internal to the terminal. Thoughts?

Reply:
Yes, I agree. It'll take a bit more shuffling but I can make that happen.

Reply:
Yeah, this wouldn't have worked as is. Oops!